Conan.VisualStudio/ConanOptionsPage.cs

```diff
@@ -14,13 +16,25 @@ public class ConanOptionsPage : DialogPage
         private ConanBuildType? _conanBuild;
         private bool? _conanUpdate;
 
+
+        private bool ValidateConanExecutableAndShowMessage(string exe)
+        {
+            if (!ConanPathHelper.ValidateConanExecutable(exe, out string errorMessage))
+            {
+                MessageBox.Show(errorMessage, "invalid conan executable",
```

Comment:
```suggestion
                MessageBox.Show(errorMessage, "Invalid conan executable",
```

Uppercase first letter, I think it is nicer


![image](https://user-images.githubusercontent.com/1406456/61301816-008b7680-a7e5-11e9-9d8c-73a68bf4301d.png)